### ISSUE TITLE
UI Bug: Active command line behind toolbar

### ISSUE THREAD

**Issue by JanJastrow** [holzschu/a-shell#180]
Hi,
I have the problem with the UI.
When using the „floating“ keyboard on iPadOS or an external keyboard (on iPhone or iPad), the active line sometimes get‘s pushed behind the toolbar (see screenshot).
This makes it hard to see what you are actually typing or see the result of you command.

The only way „out“ of this that I have found, is to disable the floating keyboard again (or disable the BT keyboard), show the full-size touch keyboard, and then reverse it again.

Screenshot:
![Screenshot](https://imgur.com/download/E2et34O/A-shell)

**Comment by holzschu** [comment]
Yes, I've noticed the same thing. It also happens sometimes when switching to an external keyboard. The problem is that it's an iOS bug, so it's a bit more difficult to solve. I'll have a look anyway.
One thing that works for me is to close and reopen the app, by clicking on the home button, then re-clicking on the app.
